cebra/integrations/sklearn/cebra.py

```diff
@@ -1216,15 +1335,41 @@ def save(self, filename: str, backend: Literal["torch"] = "torch"):
             >>> cebra_model.save('/tmp/foo.pt')
```

Comment:
really no other inputs?

Reply:
this was how the previous version behaved. we can discuss what output we should give to user

Reply:
yes, no other inputs.